src/app/(auth)/login/page.js

```diff
@@ -49,9 +50,12 @@ export default function Login() {
             <FormikTextField name="email" label="Email Address" fullWidth sx={{ mt: 3 }} />
             <FormikTextField name="password" label="Password" type="password" fullWidth sx={{ mt: 1.5 }} />
             <FormikErrors sx={{ mt: 1.5 }} />
-            <Button type="submit" variant="contained" color="green" fullWidth sx={{ mt: 6 }}>
+            <Button type="submit" variant="contained" color="green" fullWidth sx={{ mt: 6, mb: 2 }}>
               Login
             </Button>
+              <Link href="/signup" sx={{ textAlign: 'center', fontWeight: 'bold', color: 'primary.main' }}>
```

Comment:
do you need this style here why fontWeight: 'bold', color: 'primary.main'? Now this style is still not applied in the function to which you pass them. Just delete it, it looks better without them to me